app/src/main/java/by/evgen/an11pazniakhomework14/view/MainActivity.kt

```diff
@@ -0,0 +1,286 @@
+package by.evgen.an11pazniakhomework14.view
+
+import android.os.Bundle
+import android.widget.Button
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.ContextCompat
+import by.evgen.an11pazniakhomework14.R
+import by.evgen.an11pazniakhomework14.homework15.data.*
+import by.evgen.an11pazniakhomework14.homework15.observer.Editor
+import by.evgen.an11pazniakhomework14.homework15.observer.UpdateDrinksListener
+import by.evgen.an11pazniakhomework14.homework15.util.CatalogClass
+import by.evgen.an11pazniakhomework14.homework15.util.OrderList
+import by.evgen.an11pazniakhomework14.homework15.util.getInfo
+import by.evgen.an11pazniakhomework14.old.CatDataClass
+import by.evgen.an11pazniakhomework14.old.createNewBarsik
+
+class MainActivity : AppCompatActivity() {
+    private lateinit var result: String
+    private lateinit var drinkTextView: TextView
+    private lateinit var catalogTextView: TextView
+    private var iter = 0
+    private var catalogClass: CatalogClass<Spirits> = CatalogClass()
+//    private var isFirstClick = true
```

Comment:
remove comments

Reply:
Comments in task-16 was necessary code for 14 and 15 tasks. Functionality of 14 and 15 tasks not used at task 16 (example - add buttons in task 15, remove buttons in task 16). One of the condition of the tasks was to improve our app, not to write new. That is why I did not remove this code, only commented before commit and push. 
Now i fixed it and removed all comments. Thank you.